core/javatests/com/google/instrumentation/stats/TagTest.java

```diff
@@ -31,6 +31,29 @@ public void testTagTest() {
     assertThat(tag.getValue()).isEqualTo(V1);
   }
 
+  @Test
+  public void testEquals() {
+    assertThat(Tag.create(K1, V1)).isEqualTo(Tag.create(K1, V1));
+    assertThat(Tag.create(K1, V1)).isNotEqualTo(Tag.create(K2, V1));
+    assertThat(Tag.create(K1, V1)).isNotEqualTo(Tag.create(K1, V2));
```

Comment:
EqualsTester can also test that Objects unequal.  We use it for a similar test in TagKeyTest: https://github.com/google/instrumentation-java/blob/2e621e42394633c0a4a75cb59d54e176f66b646e/core/javatests/com/google/instrumentation/stats/TagKeyTest.java#L49